assets/js/templates/search-results.handlebars

```diff
@@ -1,31 +1,33 @@
 <h1>Search Results for <em>{{value}}</em></h1>
 
 {{#if empty}}
-  <p>Sorry, we couldn't find anything for <em>{{value}}</em>.</p>
+<p>Sorry, we couldn't find anything for <em>{{value}}</em>.</p>
 {{else}}
-  {{#each levels}}
-    <h2 class="results-title">{{name}}</h2>
-    {{#each results}}
-      <div class="result">
-        <h3 class="result-id">
-          <a href="{{id}}.html">{{{match}}}</a>
-        </h3>
-        <ul class="functions">
-          {{#each functions}}
-            <li class="result-elem"><a href="{{../id}}.html#{{{anchor}}}">{{{match}}}</a></li>
-          {{/each}}
-        </ul>
-        <ul class="callbacks">
-          {{#each callbacks}}
-            <li class="result-elem"><a href="{{../id}}.html#{{{anchor}}}">{{{match}}}</a> (callback)</li>
-          {{/each}}
-        </ul>
-        <ul class="types">
-          {{#each types}}
-            <li class="result-elem"><a href="{{../id}}.html#{{{anchor}}}">{{{match}}}</a> (type)</li>
-          {{/each}}
-        </ul>
-      </div>
+{{#each levels}}
+<h2 class="results-title">{{name}}</h2>
+{{#each results}}
+<div class="result">
+  <h3 class="result-id">
+    <a href="{{id}}.html">{{{match}}}</a>
+  </h3>
+  <ul class="functions">
+    {{#each functions}}
+    <li class="result-elem"><a href="{{../id}}.html#{{{anchor}}}" title="{{../id}}.{{anchor}}">{{{match}}}</a></li>
```

Comment:
Is this whole re-indent actually needed for this feature? It makes a bit difficult to see the _real_ differences unless you ignore the whitespace changes. In the future try to accommodate to the current style of send another commit just doing the re-indent.

Reply:
Completely unintentional spacing changes. I neglected to undo the changes from my formatter. I'll change these and comment again once it is fixed up.